src/patterns/task-list-pages/index.md.njk

```diff
@@ -18,7 +18,7 @@ Task list pages help users understand:
 
 ![A screenshot showing an example of the task list page, includes a heading, and three grouped sections that each contain tasks, some of these tasks are marked as completed.](task-list-whole.png)
 
-There is a [coded example of a task list page](https://govuk-prototype-kit.herokuapp.com/docs/templates/task-list) in the GOV.UK Prototype Kit.
+There is a coded example of a task list page in the [GOV.UK Prototype Kit's task list template](https://govuk-prototype-kit.herokuapp.com/docs/templates/task-list). To get the code, right-click anywhere on the Kit template and then select **View Page Source**. Alternatively, you can get the code from the [HTML file in our Prototype Kit repository on GitHub](https://github.com/alphagov/govuk-prototype-kit/blob/main/docs/views/templates/task-list.html#L18).
```

Comment:
Wonder if there's an agnostic way of telling users how to view the page source (because what option they see will depend on their browser)...

Reply:
Or maybe we could just direct users to our GitHub repo?